pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -0,0 +1,116 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-nfsplugin-provisioner
```

Comment:
can we keep the name just as `csi-nfs-provisioner`

Reply:
> can we keep the name just as `csi-nfs-provisioner`

I'd like to keep it as `csi-nfsplugin-provisioner`, it is similar to `csi-rbdplugin-provisioner` and `csi-cephfsplugin-provisioner` and is called the same [here](https://github.com/ceph/ceph-csi/blob/devel/deploy/nfs/kubernetes/csi-nfsplugin-provisioner.yaml#L20) too.